src/lib/crypto/kimchi_bindings/stubs/src/pasta_fq_plonk_index.rs

```diff
@@ -39,6 +42,7 @@ impl ocaml::custom::Custom for CamlPastaFqPlonkIndex {
 pub fn caml_pasta_fq_plonk_index_create(
     gates: CamlPastaFqPlonkGateVectorPtr,
     public: ocaml::Int,
+    runtime_tables: Vec<CamlRuntimeTableCfg<CamlFq>>,
```

Comment:
Just curious why not an Option?

Reply:
No strong reason. As the empty list can represent `None`, I thought using a vec directly would be cleaner. Happy to change if you want. I am fine with both.